packages/Switch/src/Switch.styles.js

```diff
@@ -8,36 +8,33 @@ const KnobPositionStyles = {
     transform: translateX(12px);
   `,
   medium: css`
-    transform: translate(16px, 0);
+    transform: translate(20px, 0);
   `,
   large: css`
-    transform: translate(20px);
+    transform: translate(28px);
```

Comment:
Not sure why these changed... maybe a conflict resolution with integration after you made styling changes to the Switch? Be careful not to undo your previous changes 😬